Gemfile

```diff
@@ -201,7 +201,7 @@ gem 'rubyzip', require: 'zip'
 gem 'nokogiri', '>= 1.6.8'
 
 # Polyglot support
-gem 'coursemology-polyglot', '>= 0.1.0'
+gem 'coursemology-polyglot', github: 'Coursemology/polyglot'
```

Comment:
@zhengwei143 Should use the version of ruby gems instead. I have released the latest.